components/artwork_item/templates/partials/bid_status.jade

```diff
@@ -0,0 +1,36 @@
+unless auction.isAuctionPromo() && !auction.isAuctionPromoInquirable()
+  .aga-bidding
```

Comment:
🤔  with React in the mix now we could potentially go all in and use it in place of jade templates. Not saying we _should_ do that here, but food for thought in the next greenfield app.